cddl/measurement-values-map.cddl

```diff
@@ -1,19 +1,19 @@
 measurement-values-map = non-empty<{
-  ? &(version: 0) => version-map
-  ? &(svn: 1) => svn-type-choice
-  ? &(digests: 2) => digests-type
-  ? &(flags: 3) => flags-map
+  ? &(version: 0) => any-of<version-map>
```

Comment:
If we decide to introduce `any-of` semantics, then we should also provide a comparison logics for this semantics, for each one of the elements inside the value map.

While this is not a gating criterion for progressing this PR, it might be worth noting this using Edit on an already opened github issue, or creating a new one, if absent!

Reply:
Is the description in the matching operators section insufficient for defining the comparison semantics?

What do you mean comparison logics?